www/index.ejs

```diff
@@ -3,30 +3,16 @@
 <head>
   <meta charset="utf-8">
   <meta http-equiv="x-ua-compatible" content="ie=edge">
-  <title><%= config.title %></title>
   <meta name="description" content="">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="apple-touch-icon" href="/apple-touch-icon.png">
```

Comment:
Is there any favicon right now then? We have to ask neuburg probably about this.

Reply:
No currently not, but we've already asked about it.